releases/SN30_Pro+/3.02/sn30pro_plus.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Richard Hughes <richard@hughsie.com> -->
+<!-- Copyright 2019 Mario Limonciello <mario.limonciello@dell.com> -->
+<component type="firmware">
+  <id>com.8bitdo.sn30pro_plus.firmware</id>
+  <name>SN30 Pro+</name>
+  <summary>Firmware for the 8Bitdo SN30 Pro+ Game Controller</summary>
+  <description>
+    <p>
+      Updating the firmware on your SN30 Pro+ device improves performance and adds
+      new features.
+    </p>
+  </description>
+  <provides>
```

Comment:
Should I add the GUIDs for the Android mode (B+Start) USB IDs?  Like this:

```suggestion
  <provides>
    <!-- USB\VID_2DC8&PID_6002&REV_0001 -->
    <firmware type="flashed">64eb3e9c-e79c-50d0-984b-21a1641f37c5</firmware>
    <!-- USB\VID_2DC8&PID_6002 -->
    <firmware type="flashed">a9a82ee0-7fdc-5d80-bd73-6b39f7a4f061</firmware>
```

Reply:
I'm hesitant to add them if the controller doesn't seem to work with fwupd in this mode.

Reply:
OK